### PR TITLE
Added additional faq data

### PR DESCRIPTION
**Pull request by codedbychavez**
### Description

Added answers to the following questions:

- Can I sign a data processing agreement with ConfigCat?
- Which data is processed and stored from browsers accessing ConfigCat's CDN to pull feature toggles?
- Is ConfigCat's data only hosted within the EU?
- Is it possible to export feature flags in ConfigCat?
- How can I use AND operators in my targeting rules in ConfigCat?

Shifted the following question up for better alignment with the faq page:
- How can I be sure that my data is safe at ConfigCat?

### Requirement checklist

- [x] I have validated my changes on a test/local environment.
- [ ] I have checked the SNYK/Dependabot reports and applied the suggested changes.
- [ ] (Optional) I have updated outdated packages.

### CODE REVIEW

File: website/src/schema-markup/faq.json [hunk 1]
```diff
@@ -156,34 +156,66 @@
...
             }
         },
         {
             "@type": "Question",
-            "name": "Are ConfigCat percentage targeting options sticky?",
+            "name": "Can I sign a data processing agreement with ConfigCat?",
             "acceptedAnswer": {
                 "@type": "Answer",
-                "text": "Yes. The percentage-based targeting is sticky by design and consistent across all SDKs.  Also, consider the following:  - All SDKs evaluate the rules in the exact same way. (10% is the same 10% in all SDKs)  - The percentage rules are sticky by feature flag. (10% is a different 10% for each feature flag)  More on stickiness and consistency: https://configcat.com/docs/advanced/targeting/#stickiness--consistency"
+                "text": "Yes, ConfigCat have no trouble signing a data processing agreement (DPA)"
```
Comment (sigewuzhere) [inline website/src/schema-markup/faq.json:170]:
```suggestion
                "text": "Yes, ConfigCat is happy to sign a data processing agreement (DPA)"
```

Please update the original one as well

File: website/src/schema-markup/faq.json [hunk 2]
```diff
@@ -156,34 +156,66 @@
...
             }
         },
         {
             "@type": "Question",
-            "name": "How can I be sure that my data is safe at ConfigCat?",
+            "name": "Which data is processed and stored from browsers accessing ConfigCat's CDN to pull feature toggles?",
             "acceptedAnswer": {
                 "@type": "Answer",
-                "text": "The feature flag evaluation is done on the client-side in the ConfigCat SDK. So if you are using the SDK, you can be sure that your data will never leave your system.  See ConfigCat's architecture explained here: https://configcat.com/architecture/."
+                "text": "ConfigCat don't store anything from browsers. The data flow is one directional - the SDKs are only downloading the config.json files and the feature flag evaluation happens in the SDK. ConfigCat doesn't know anything about your users, only those data which you set in your targeting rules."
```
Comment (sigewuzhere) [inline website/src/schema-markup/faq.json:178]:
```suggestion
                "text": "ConfigCat doesn't store any browser data. The data flow is one directional - the SDKs are only downloading the config.json files and the feature flag evaluation happens in the SDK. ConfigCat doesn't know anything about your users, only those data which you set in your targeting rules."
```
 Please update the original

File: website/src/schema-markup/faq.json [hunk 3]
```diff
@@ -156,34 +156,66 @@
...
             "acceptedAnswer": {
                 "@type": "Answer",
-                "text": "Yes. The percentage-based targeting is sticky by design and consistent across all SDKs.  Also, consider the following:  - All SDKs evaluate the rules in the exact same way. (10% is the same 10% in all SDKs)  - The percentage rules are sticky by feature flag. (10% is a different 10% for each feature flag)  More on stickiness and consistency: https://configcat.com/docs/advanced/targeting/#stickiness--consistency"
+                "text": "Yes, ConfigCat have no trouble signing a data processing agreement (DPA)"
             }
         },
         {
             "@type": "Question",
-            "name": "How can I be sure that my data is safe at ConfigCat?",
+            "name": "Which data is processed and stored from browsers accessing ConfigCat's CDN to pull feature toggles?",
```
Comment (sigewuzhere) [inline website/src/schema-markup/faq.json:175]:
```suggestion
            "name": "What data is stored or processed from browsers accessing ConfigCat's CDN when pulling feature toggles?",
```
 Please update the original

File: website/src/schema-markup/faq.json [hunk 4]
```diff
@@ -156,34 +156,66 @@
...
+                "@type": "Answer",
+                "text": "If you want to use targeting rules based on email address, phone number, or other sensitive data, you can use the Confidential text comparators: https://configcat.com/docs/advanced/targeting/#confidential-text-comparators."
+            }
+        },
+        {
+            "@type": "Question",
+            "name": "Are ConfigCat percentage targeting options sticky?",
+            "acceptedAnswer": {
+                "@type": "Answer",
+                "text": "Yes. The percentage-based targeting is sticky by design and consistent across all SDKs.  Also, consider the following:  - All SDKs evaluate the rules in the exact same way. (10% is the same 10% in all SDKs)  - The percentage rules are sticky by feature flag. (10% is a different 10% for each feature flag)  More on stickiness and consistency: https://configcat.com/docs/advanced/targeting/#stickiness--consistency"
```
Comment (sigewuzhere) [inline website/src/schema-markup/faq.json:218]:
```suggestion
                "text": "Yes. The percentage-based targeting is sticky by design and consistent across all SDKs. More on stickiness and consistency: https://configcat.com/docs/advanced/targeting/#stickiness--consistency"
```

**Review by sigewuzhere — CHANGES_REQUESTED** [review]

**Review by sigewuzhere — CHANGES_REQUESTED** [review]
I think you missed a note in my previous comments about updating the original answers.

**Comment by codedbychavez** [comment]
> I think you missed a note in my previous comments about updating the original answers.

@sigewuzhere Did you mean this [https://github.com/configcat/docs/pull/184](https://github.com/configcat/docs/pull/184) ? or maybe I misunderstood. Feel free to let me know.

**Comment by sigewuzhere** [comment]
> > I think you missed a note in my previous comments about updating the original answers.
> 
> 
> 
> @sigewuzhere Did you mean this [https://github.com/configcat/docs/pull/184](https://github.com/configcat/docs/pull/184) ? or maybe I misunderstood. Feel free to let me know.

Yes those are the ones. If you get comments in a PR, please fix them there. Now, it is hard to see what belongs where.

**Comment by codedbychavez** [comment]
> > > I think you missed a note in my previous comments about updating the original answers.
> > 
> > 
> > @sigewuzhere Did you mean this #184 ? or maybe I misunderstood. Feel free to let me know.
> 
> @codedbychavez Yes those are the ones. Please add them here and close the other PR, it is hard to track what belongs where.

Alright, will do.

**Comment by codedbychavez** [comment]
@sigewuzhere Done. You can take another look.

**Review by sigewuzhere — DISMISSED** [review]

**Comment by codedbychavez** [comment]
@sigewuzhere I had to merge master into this branch to resolve merge conflicts. Requesting your review again.

**Review by sigewuzhere — APPROVED** [review]

**Pull request merged**